### DISCUSSION TITLE
v2中如何局部修改schema

### DISCUSSION THREAD

**Discussion by NexxLuo** [alibaba/formily#1291] [Q&A 2.x] [answered]
json schema模式下部分场景需要动态修改schema，比如：一个点击一个按钮，给一个Tab组件添加TabPane，当然可以选择自定义Tab组件通过控制字段模型来处理，但是这样就无法同步到schema中。

**Comment by janryWang** [comment, marked as answer]
不支持局部修改，要修改只能全局修改
